test/service/test_jobs.py

```diff
@@ -73,46 +73,46 @@ def create_mock(self, target_name):
         return patcher.start()
 
     def test_returns_job(self):
-        self.mock_get_algo.return_value = create_algorithm()
+        self.mock_get_algo.return_value = create_algorithm('pzsvc-ndwi-py')
         self.mock_get_scene.return_value = create_scene()
         self.mock_requests.post('/tides', text=RESPONSE_TIDE)
         job = jobs.create(API_KEY, 'test-user-id', 'test-scene-id', 'test-service-id', 'test-name')
         self.assertIsInstance(job, jobs.Job)
 
     def test_assigns_correct_algorithm_name(self):
-        self.mock_get_algo.return_value = create_algorithm()
+        self.mock_get_algo.return_value = create_algorithm('pzsvc-ndwi-py')
```

Comment:
`pzsvc-ndwi-py` is the default value for the `create_algorithm` function.  You don't need to repeat it here.